### PR TITLE
Fix 'days_ago' not properly display

### PR DESCRIPTION
**Pull request by zunnurs01**
Fix and add example for 'days_ago' modifier.

### CODE REVIEW

File: content/collections/modifiers/days_ago.md [hunk 1]
```diff
@@ -8,16 +8,23 @@ title: 'Days Ago'
 ---
 Returns the number of days since a given date variable. Statamic will attempt to parse any string as a date, but try to keep it in the least ambiguous date format possible.
 
+
 ```yaml
-date: October 1 2015
+//Example previous date
```
Comment (Konafets) [inline content/collections/modifiers/days_ago.md:13]:
```suggestion
// Example previous date
```

File: content/collections/modifiers/days_ago.md [hunk 2]
```diff
@@ -8,16 +8,23 @@ title: 'Days Ago'
...
+date: December 8 2021
 ```
 
-{{ noparse }}
 ```
+//Example current date on your server:
```
Comment (Konafets) [inline content/collections/modifiers/days_ago.md:18]:
```suggestion
// Example current date on your server:
```

File: content/collections/modifiers/days_ago.md [hunk 3]
```diff
@@ -8,16 +8,23 @@ title: 'Days Ago'
...
+December 12 2021
+```
+
+```antlers
+//Antlers Frontend
```
Comment (Konafets) [inline content/collections/modifiers/days_ago.md:23]:
```suggestion
// Antlers Frontend
```

File: content/collections/modifiers/days_ago.md [hunk 4]
```diff
@@ -8,16 +8,23 @@ title: 'Days Ago'
...
 {{ date | days_ago }}
 ```
-{{ /noparse }}
 
 ```html
-{{ test_date | days_ago }}
+//Output:
```
Comment (Konafets) [inline content/collections/modifiers/days_ago.md:28]:
```suggestion
// Output:
```

**Review by Konafets — CHANGES_REQUESTED** [review]

**Comment by Konafets** [comment]
Tried the changes and beside the missing space, it looks fine for me. It also removes the `{{ noparse }}` output.

**Comment by jackmcdade** [comment]
Thanks!

**Pull request merged**